components/brave_vpn/browser/connection/win/utils.cc

```diff
@@ -229,6 +234,9 @@ RasOperationResult DisconnectEntry(const std::wstring& entry_name) {
         if (name.compare(entry_name) == 0 && type.compare(L"VPN") == 0) {
           VLOG(2) << __func__ << " : Disconnect... " << entry_name;
           dw_ret = RasHangUpA(lp_ras_conn[i].hrasconn);
+          if (dw_ret != ERROR_SUCCESS) {
+            caller = "RasHangUpA()";
```

Comment:
Just noticed above `RasHangUpA` usage - mixing usages of the A and W functions can cause a problem. By default, all the Win32 API calls you can leave the A (ascii) or W (wide char) off the end and it'll pick the right ones at compile time

Can we change above to `RasHangUp` (and then this line we can change to `caller = "RasHangUp()";`

Reply:
Fixed